tests/unit_tests/test_lobbyconnection.py

```diff
@@ -957,6 +957,10 @@ async def test_command_game_matchmaking_not_party_owner(
 
     lobbyconnection.ladder_service.cancel_search.assert_called_once()
 
+async def test_command_match_ready(lobbyconnection):
```

Comment:
Please put 2 blank lines between functions, otherwise my linter will yell at me.

Reply:
*Top level functions only. If they're methods of a class then only 1 line